src/main/java/org/opentripplanner/analyst/ResultSetDelta.java

```diff
@@ -2,7 +2,7 @@
 
 import java.util.Map.Entry;
 
-public class ResultSetDelta extends ResultSetWithTimes {
+public class ResultSetDelta extends ResultSet {
```

Comment:
This class needs some javadoc. It's possible to guess from the name what it does, but where it's used, why, and how it works are less obvious.